src/pythermalcomfort/models.py

```diff
@@ -1653,10 +1667,10 @@ def ankle_draft(tdb, tr, vr, rh, met, clo, v_ankle, units="SI"):
     --------
     .. code-block:: python
 
-        >>> from pythermalcomfort.models import ankle_draft
-        >>> results = ankle_draft(25, 25, 0.2, 50, 1.2, 0.5, 0.3, units="SI")
-        >>> print(results)
-        {'PPD_ad': 18.5, 'Acceptability': True}
+    >>> from pythermalcomfort.models import ankle_draft
```

Comment:
could you please remove these formatting changes? I formatted the documentation with a Python package and unless is creating issues I would like to keep it as it is. If you reformatted since you spotted some issues in the docs please let me know and I am happy to change it.